src/server/service/kv.rs

```diff
@@ -41,6 +41,7 @@ use util::future::paired_future_callback;
 use util::worker::Scheduler;
 
 const SCHEDULER_IS_BUSY: &str = "scheduler is busy";
+const GC_WORKER_IS_BUSY: &str = "tikv's gc worker is busy";
```

Comment:
"gc worker is busy" should be enough.

Reply:
Since there is a module named `gc_worker` in TiDB too, I'm afraid it may be confusing

Reply:
ok

Reply:
remove `tikv's`, redundancy.

Reply:
This message will be sent to TiDB, and there is a module named `gc_worker` in TiDB too. It may produce confusing logs in TiDB.

Reply:
Ok I think it is in fact TiDB's job to differentiate and mark messages from TiKV explicitly, for example, adding "[TiKV Error]" as a prefix. For us, we should not take care what TiDB has.